components/admin_console/permission_schemes_settings/guest_permissions_tree/guest_permissions_tree.jsx

```diff
@@ -55,8 +56,11 @@ export default class GuestPermissionsTree extends React.Component {
                 ],
             },
             Permissions.USE_CHANNEL_MENTIONS,
-            Permissions.USE_GROUP_MENTIONS,
         ];
+
+        if (props.license.LDAPGroups === 'true') {
```

Comment:
```suggestion
        if (this.props.license && this.props.license.IsLicensed === 'true' && this.props.license.LDAPGroups === 'true') {
```
Maybe a little bit more precise if done this way? It's how I noticed it was done Catalin's PR and I thought it could be more accurate check (https://github.com/mattermost/mattermost-webapp/pull/5382/files#diff-1d27e4a784ea64f677a61481b5585a94R233)